### PR TITLE
Dissolved /enterprise/ dir, made TOC titles match top-of-page title

### PR DESCRIPTION
**Pull request by mikehoffms**
**The four moved .md files/pages, rendered, in order of existing TOC:**
* https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/declare-permissions?branch=pr-en-us-1985 - resided in /enterprise/ dir but was presented in Developer Guide TOC bucket & remains there.
* https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/match-patterns?branch=pr-en-us-1985 - moved to Developer Guide TOC bucket, **OK?**

* These two pages moved from /enterprise/ dir to /publish/ dir, and moved from Enterprise TOC bucket (which is now gone) to Publish TOC bucket next to other Publish articles, **OK?**
   * https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/hosting-and-updating?branch=pr-en-us-1985
   * https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/auto-update?branch=pr-en-us-1985

**The redirects to test:**
* https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/enterprise/declare-permissions?branch=pr-en-us-1985 - passed.
* https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/enterprise/match-patterns?branch=pr-en-us-1985 - passed.
* https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/enterprise/hosting-and-updating?branch=pr-en-us-1985 - passed.
* https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/enterprise/auto-update?branch=pr-en-us-1985 - passed.

### CODE REVIEW

**Review by vchapel — APPROVED** [review]
LGTM

File: microsoft-edge/toc.yml [hunk 1]
```diff
@@ -816,77 +815,75 @@
...
-            - name: Update an extension
+            - name: Publish and update extensions in the Microsoft Edge Add-ons website
+              href: extensions-chromium/publish/hosting-and-updating.md  # moved from /enterprise/
+
+            - name: Update a Microsoft Edge extension
               href: extensions-chromium/publish/update-extension.md
 
-            - name: Submission states
+            - name: Automatically update extensions in Microsoft Edge
+              href: extensions-chromium/publish/auto-update.md  # moved from /enterprise/
```
Comment (mikehoffms) [inline microsoft-edge/toc.yml:853]: @sonakshimore See Note at top of the upstream doc, https://developer.chrome.com/docs/apps/autoupdate/

File: microsoft-edge/toc.yml [hunk 2]
```diff
@@ -816,77 +815,75 @@
...
             - name: Add users to the Microsoft Edge program
               href: extensions-chromium/publish/aad-account.md
 
-            - name: Publish an extension
+            - name: Publish a Microsoft Edge extension
               href: extensions-chromium/publish/publish-extension.md
 
-            - name: Update an extension
+            - name: Publish and update extensions in the Microsoft Edge Add-ons website
+              href: extensions-chromium/publish/hosting-and-updating.md  # moved from /enterprise/
```
Comment (mikehoffms) [inline microsoft-edge/toc.yml:847]: @ashishnirvana @sonakshimore - Dissolve hosting-and-updating.md; merge its content into other Publish article? rendered:  https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/hosting-and-updating?branch=pr-en-us-1985

**Comment by sonakshimore** [comment]
Looks good to me.

@mikehoffms what about the conflicting files?

File: microsoft-edge/extensions-chromium/publish/auto-update.md [hunk 3]
```diff
@@ -22,18 +22,16 @@ ms.date: 04/13/2021
    limitations under the License.  -->
 # Automatically update extensions in Microsoft Edge
 
-When you set your extension to automatically update, your extension shares the following benefits with Microsoft Edge when set to automatically update.
+> [!NOTE]
+> This article doesn't apply to extensions that you publish using the [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) dashboard.  You can use the dashboard to release updated versions to your users and to the Microsoft Edge Add-ons website.  For more information, see [Update a Microsoft Edge extension](../publish/update-extension.md).
```
Comment (ashishnirvana1) [inline microsoft-edge/extensions-chromium/publish/auto-update.md:26]: we can merge this document for now, but separately we should review this document with engineering team for its relevance.

File: microsoft-edge/toc.yml (hunk 1 above)
Comment (ashishnirvana1) [reply microsoft-edge/toc.yml:850]: Have asked an engineering team mate to review this document. https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/auto-update?branch=pr-en-us-1985

But I recommend we go ahead with merging this - and sharing back the published document link as soon as its merged. We can take his inputs into consideration later.

**Review by ashishnirvana1 — APPROVED** [review]

**Pull request merged**